app/controllers/schedules_controller.rb

```diff
@@ -52,9 +52,11 @@ def load_current_guider
   def schedule_json
```

Comment:
This doesn't feel like a controller concern now. Can we move it to a dedicated serialiser?